subhosting/manual/subhosting.md

```diff
@@ -0,0 +1,124 @@
+# Subhosting Concepts
+
+To build Subhosting with Deno Deploy, it helps to understand some key resources
+within the system. These resources are also represented in the
+[REST API](../api/index.md).
+
+![overview of subhosting resources](./subhosting-org-structure.svg)
+
+<a style={{marginTop: '4rem'}} id="organizations"></a>
+
+## Organizations
+
+[**Organizations**](https://apidocs.deno.com/#get-/organizations/-organizationId-)
+are a container for all data related to a subhosting implementation. Your
+organization will have a name and an ID. Each organization has an analytics
+endpoint which can be used to get metrics (such as request count and bandwidth
+used) from across the organization.
+
+Other Deploy users can be invited to collaborate on an organization, and
+[access tokens](https://dash.deno.com/account#access-tokens) can give developers
+with organization access the ability to modify resources within the org via API.
+New organizations can be created in the
+[Deploy dashboard](https://dash.deno.com/orgs/new).
+
+<a style={{marginTop: '4rem'}} id="deployments"></a>
+
+## Deployments
```

Comment:
FYI projects come before deployments in the side nav but the content of deployments comes before projects here. I imagine the sidenav is the right one given it follows the nesting narrative?